osgar/drivers/kloubak.py

```diff
@@ -266,8 +266,9 @@ def __init__(self, config, bus):
         self.last_pose_encoders = [0, 0,] * self.num_axis  # accumulated tacho readings
         self.encoders = [0, 0,] * self.num_axis  # handling 16bit integer overflow
         self.last_encoders_16bit = None  # raw readings
-        self.last_join_angle = None
-        self.last_join_angle2 = None
+        self.last_join_angle = None  # join angle in radians
+        self.last_join_angle2 = None  # # join2 angle in radians
+        self.prev_joins_angle = None  # angle between front and rear axle, used for heading
```

Comment:
`prev` is typically "previous" in a sequence

Reply:
Yes, it is previous joint angle (for K3 it is a sum of the first and the second joint angle) which was measured during previous cycle.